build.gradle

```
@@ -0,0 +1,158 @@
+        // Debug builds will always be signed. If no environment variables are set, a default
+        // keystore will be initialized by the task initDebugKeystore and used. This keystore
+        // can be uploaded to GitHub secrets by following CONTRIBUTING.md#Using the local debug.keystore on the Github CI actions
+        // in order to always receive correctly signed debug APKs from the CI.
+        debug {
+            storeFile (System.env.DEBUG_KEYSTORE ? file(System.env.DEBUG_KEYSTORE) : file("debug.keystore"))
+            storePassword (System.env.DEBUG_KEYSTORE_PASSWORD ? "$System.env.DEBUG_KEYSTORE_PASSWORD" : "debug0")
+            keyAlias (System.env.DEBUG_KEY_ALIAS ? "$System.env.DEBUG_KEY_ALIAS" : "debug")
+            keyPassword (System.env.DEBUG_KEY_PASSWORD ? "$System.env.DEBUG_KEY_PASSWORD" : "debug0")
+        }
+        // Release builds will only be signed with a dedicated key if specified. This key has to be
+        // created manually. To be used with GitHub CI, key and environment variables have to be
+        // added as secrets. See file .github/workflows/make-apk.yml for further reference.
+        release {
+            if (System.env.RELEASE_KEYSTORE) {
+                storeFile file(System.env.RELEASE_KEYSTORE)
+                storePassword "$System.env.RELEASE_KEYSTORE_PASSWORD"
+                keyAlias "$System.env.RELEASE_KEY_ALIAS"
+                keyPassword "$System.env.RELEASE_KEY_PASSWORD"
+            }
+        }
+    }
+    buildTypes {
+        release {
+            minifyEnabled true
+            shrinkResources true
+            debuggable false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt')
+            resValue "string", "app_name", "@string/app_name_release"
+            if (System.env.RELEASE_KEYSTORE) {
+                signingConfig signingConfigs.release
+            }
+        }
+        debug {
+            minifyEnabled false
+            shrinkResources false
+            debuggable true
+            applicationIdSuffix ".debug"
+            resValue "string", "app_name", "@string/app_name_debug"
+            resValue "bool", "debug_logs", "true"
+            signingConfig signingConfigs.debug
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_7
+        targetCompatibility JavaVersion.VERSION_1_7
+    }
+}
+
+dependencies {
+
+}
+
+tasks.register('buildKeyboardFont') {
+    println "Executing fontforge to build custom font\n"
+    mkdir "$buildDir"
+    new ByteArrayOutputStream().withStream { bos ->
+        exec {
+            workingDir "$projectDir/special_font"
+            def svgFiles = workingDir.listFiles().findAll {
+                it.isFile() && it.name.endsWith(".svg")
+            }
+
+            commandLine("fontforge", "-lang=ff", "-script", "build.pe", "$buildDir/special_font.ttf", *svgFiles)
+            standardOutput = bos
+        }
+        println "$bos"
+    }
```

```suggestion
    exec {
        workingDir "$projectDir/special_font"
        def svgFiles = workingDir.listFiles().findAll {
            it.isFile() && it.name.endsWith(".svg")
        }

        commandLine("fontforge", "-lang=ff", "-script", "build.pe", "$buildDir/special_font.ttf", *svgFiles)
    }
```
We can leave standard output alone. Same for all the others except `checkKeyboardLayouts`.